src/main/java/net/dries007/tfc/api/util/IPlaceableItem.java

```diff
@@ -73,7 +73,7 @@ default int consumeAmount()
 
                 IBlockState state = world.getBlockState(pos);
 
-                if (facing == EnumFacing.UP && state.getBlock() == BlocksTFC.CHARCOAL_PILE)
```

Comment:
What was the motivation for this change?
For reference, it is a fix for this (not TFC, but basically identical) bug: https://github.com/alcatrazEscapee/tinkers-forging/issues/15